static/css/legacy.less

```diff
@@ -1662,8 +1662,8 @@ span.actions {
 // openlibrary/templates/account/books.html
 // openlibrary/macros/SearchResults.html
 // openlibrary/macros/FulltextResults.html
-// why not form#siteSearch ?
-ul#siteSearch {
+// why not form.list-books ?
```

Comment:
this comment no longer makes sense and can be removed.
We now understand there is a form#siteSearch and a ul#siteSearch but that is no longer the case.